changelog.md

```diff
@@ -1,3 +1,7 @@
+## Changes in version 0.?.?.?
```

Comment:
There's already a "Changes in version 0.?.?.?" section below. Put the mention of `unsafeInterleave` in that section instead.

Reply:
The likely reason for confusion is that we already have two, one for a bug-fix release and one for the next major or minor release.

Reply:
Oh dear, I didn't notice that! In that case, I'm not exactly sure myself where this mention of `unsafeInterleave` should go...